### PR TITLE
Show 'no known risks' when risks section empty

### PR DESCRIPTION
**Pull request by doctorpod**
Ticket: https://trello.com/c/dNDPcXhV

### CODE REVIEW

File: app/models/concerns/questionable.rb [hunk 1]
```diff
@@ -89,5 +89,9 @@ def mandatory_questions
     def all_questions_answered?
       mandatory_questions.all?(&:answered?)
     end
+
+    def any_questions_answered?
```
Comment (cesidio) [inline app/models/concerns/questionable.rb]: What about?:

```
   def no_questions_answered?		
      mandatory_questions.none?(&:answered?)		
    end
```

Comment (doctorpod) [reply app/models/concerns/questionable.rb]: Have refactored the body of the method to more closely match your example but kept the name the same. Calling `any_questions_answered?` from the view is more readable than `!no_questions_answered?`.

**Review by cesidio — APPROVED** [review]
👍

File: app/views/escorts/_assessment.html.slim [hunk 2]
```diff
@@ -12,7 +12,7 @@ section id=name.downcase
       p.review
         = t('assessment.confirmed_timestamp', name: assessment.reviewer.full_name, time: assessment.reviewed_at.to_s(:humanized))
 
-  - if assessment
+  - if assessment && assessment.any_questions_relevant?
```
Comment (cesidio) [inline app/views/escorts/_assessment.html.slim]: usually rubocop complains about not using & operator
assessment&.any_questions_relevant?

**Review by cesidio — APPROVED** [review]

**Pull request merged**